lib/tether/interfaces.go

```diff
@@ -33,7 +34,7 @@ type Operations interface {
 
 	SetHostname(hostname string, aliases ...string) error
 	Apply(endpoint *NetworkEndpoint) error
-	MountLabel(ctx context.Context, label, target string) error
+	MountLabel(ctx context.Context, devicePath, target string) error
```

Comment:
Please add the type (devicePath string) there to ease reading instead of saving some keypresses.